### PR TITLE
Address issues #18

### PR DESCRIPTION
**Pull request by the-j0k3r**
Should fix first and second issues reported in #18 thx to @Mottie for opacity tip and fix for second issue :)

Im trying to figure out the third issue, so bare with me.

### CODE REVIEW

File: StylusDeepDark.user.css [hunk 1]
```diff
@@ -616,13 +616,16 @@
 	}
 
 	/*Stylus popup fixes */
+	/* Use opacity to solve disabled entries see #18 thx Mottie :)*/
 	#stylus-popup .disabled .style-name
 	{
-		color: var(--hover-background) !important;
+		opacity: .7;
```
Comment (the-j0k3r) [inline StylusDeepDark.user.css:622]: @Mottie is this opacity level OK with you?

**Review by the-j0k3r — COMMENTED** [review]
Waiting on feedback

File: StylusDeepDark.user.css (hunk 1 above)
Comment (Mottie) [reply StylusDeepDark.user.css:622]: Yeah, that's better! 😺

**Comment by Mottie** [comment]
> Im trying to figure out the third issue

Move the `:hover` to the `.disabled` selector:

```css
#stylus-popup .disabled:hover .style-name { }
```

**Comment by the-j0k3r** [comment]
Duh... Lookout! An amateur on the wheel :) 
But hey, bonus also fixed the third issue for enabled styles.

Do you think the controls can also be affected by tab'ing as is by hovering?

**Comment by Mottie** [comment]
> Do you think the controls can also be affected by tab'ing as is by hovering

Yeah, you can use the `:focus-within` for tabbing, but the controls are always bright.

**Pull request merged**